pkg/appmanager/validateResources.go

```diff
@@ -138,7 +153,13 @@ func (appMgr *Manager) checkValidIngress(
 			}
 			return false, nil
 		}
-
+		// This ensures that pool-only mode only logs the message below the first
+		// time we see a config.
+		if _, exists := appMgr.resources.GetByName(rsName); !exists &&
+			bindAddr == "" {
```

Comment:
No need to have this on a newline here, you can just keep it with the line above (unless this line is abnormally long, in which case the log message below should be broken up into shorter lines).